samples/tif/fields.tif.json

```diff
@@ -2,8 +2,13 @@
   "version": "TanaIntermediateFile V0.1",
   "attributes": [
     {
-      "name": "Owner",
+      "name": "Holder",
+      "_comment1": "the field will be ignored by Tana's TIF importer unless its name is included in the attributes array",
+      "values": [
+        "Jack"
+      ],
       "count": 1
+      "_comment2": "Tana's TIF importer will handle fields just fine without 'values' or 'count'" keys,
```

Comment:
Hey @ricardo37 

Thanks for fixing this! I tried importing the updated file, but see that the json is malformed. `keys` is outside the quotes + missing `,` on the line above. Happy to merge if you push a fix! 🚀